boards/arm/gd32f403z_eval/gd32f403z_eval_defconfig

```diff
@@ -12,3 +12,4 @@ CONFIG_CORTEX_M_SYSTICK=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
```

Comment:
Not necessary. This is selected by samples/apps.

Reply:
Done.